src/SelfServe/ClassDecorators.tsx

```diff
@@ -0,0 +1,16 @@
+import { Info } from "../Explorer/Controls/SmartUi/SmartUiComponent";
+import { addPropertyToMap, buildSmartUiDescriptor } from "./SelfServeUtils";
+
+export const IsDisplayable = (): ClassDecorator => {
+  //eslint-disable-next-line @typescript-eslint/ban-types
+  return (target: Function) => {
```

Comment:
Remove eslint rule disables and let the compiler infer the types:

```suggestion
  return (target) => {
```